integrationtests/test_signup.py

```diff
@@ -35,3 +36,40 @@ def test_signup(self):
         b.fill_in("password", "?P455W0rd!")
         b.click_button("Sign In")
         b.wait_for_element("a", text="MY WORKFLOWS", wait=True)
+
+    def test_signup_el(self):
```

Comment:
Put this in `test_i18n.py`.

Otherwise, we set a precedent: all integration-test files will could test i18n. Soon enough we get into i18n-specific minutae in each file: 'which languages should we test "Example Workflows" in?' etc.

That precedent isn't foolish; but I don't think it's the best cost/benefit for us. I prefer to let `test_i18n.py` grow as we encounter i18n-related regressions.